src/Api/AdminConsole/Authorization/Requirements/ManageAccountRecoveryRequirement.cs

```diff
@@ -0,0 +1,21 @@
+﻿#nullable enable
+
+using Bit.Core.Context;
+using Bit.Core.Enums;
+
+namespace Bit.Api.AdminConsole.Authorization.Requirements;
+
+public class ManageAccountRecoveryRequirement : IOrganizationRequirement
+{
+    public async Task<bool> AuthorizeAsync(
+        CurrentContextOrganization? organizationClaims,
+        Func<Task<bool>> isProviderUserForOrg)
+        => organizationClaims switch
+        {
+            { Type: OrganizationUserType.Owner } => true,
+            { Type: OrganizationUserType.Admin } => true,
+            { Permissions.ManageUsers: true } => true,
```

Comment:
After this change, these are fully separate permissions. So `ManageUsers` cannot manage account recovery.

```suggestion
```